blockbase/sass/base/_header.scss

```diff
@@ -1,7 +1,10 @@
 .wp-site-blocks .site-header {
+	display: flex;
 	justify-content: start;
 	overflow: inherit;
 	padding-top: var(--wp--custom--gap--vertical);
+	padding-bottom: var(--wp--custom--gap--vertical);
+	margin-bottom: var(--wp--custom--gap--vertical);
```

Comment:
In general, shouldn't all of these styles be accomplished via blocks and their style attributes?

Hardcoding CSS into these sections results in a broken FSE experience, where block controls surfaced to the user in the editor are overridden and/or have no effect when selected as a result.

Reply:
@Addison-Stavlo ideally yes, but there are things we need to do that aren't possible with the current block controls and attributes. In this case we need the site logo to appear in a different position in smaller viewports - Gutenberg doesn't yet offer us any controls for responsive layouts, so this kind of thing is not yet possible. As soon as we have the tools we need we will remove all the CSS.